### PR TITLE
[KB-248] MinMax Normalization

### PR DESCRIPTION
**Pull request by Ivikhostrup**
Is citation placement okay?

### CODE REVIEW

File: report_thesis/src/sections/background.tex [hunk 1]
```diff
@@ -5,7 +5,16 @@ \subsubsection{Standard Scaler}
 
 \subsubsection{Max Absolute Scaler}
 
-\subsubsection{MinMax Scaler}
+\subsubsection{Min-Max Normalization}
+Min-Max normalization rescales the range of features to $[0, 1]$ or $[a, b]$.
```
Comment (Pattrigue) [inline report_thesis/src/sections/background.tex:9]:
```suggestion
Min-max normalization rescales the range of features to $[0, 1]$ or $[a, b]$. 
```

File: report_thesis/src/sections/background.tex [hunk 2]
```diff
@@ -5,7 +5,16 @@ \subsubsection{Standard Scaler}
 
 \subsubsection{Max Absolute Scaler}
 
-\subsubsection{MinMax Scaler}
+\subsubsection{Min-Max Normalization}
+Min-Max normalization rescales the range of features to $[0, 1]$ or $[a, b]$. 
+The goal is to normalize the range of the data to a specific scale, typically 0 to 1. The formula for Min-Max normalization is:
```
Comment (Pattrigue) [inline report_thesis/src/sections/background.tex:10]:
```suggestion
The goal is to normalize the range of the data to a specific scale, typically 0 to 1. 
Mathematically, min-max normalization is defined as:
```

File: report_thesis/src/sections/background.tex [hunk 3]
```diff
@@ -5,7 +5,16 @@ \subsubsection{Standard Scaler}
...
+
+$$ 
+	v' = \frac{v - \min(F)}{\max(F) - \min(F)} \times (new max - new min) + new min
```
Comment (Pattrigue) [inline report_thesis/src/sections/background.tex:13]:
```suggestion
	v' = \frac{v - \min(F)}{\max(F) - \min(F)} \times (new max - new min) + new min,
```

File: report_thesis/src/sections/background.tex [hunk 4]
```diff
@@ -5,7 +5,16 @@ \subsubsection{Standard Scaler}
...
+$$
+
+where $v$ is the original value, $\min(F)$ and $\max(F)$ are the minimum and maximum values of the feature $F$ respectively, and $new min$ and $new max$ define the range of the transformed data. 
+This type of normalization is particularly useful when algorithms such as neural networks are sensitive to the magnitude of input features and require normalization to ensure more effective and faster convergence~\cite{dataminingConcepts}.
```
Comment (Pattrigue) [inline report_thesis/src/sections/background.tex:17]:
```suggestion
This type of normalization is particularly useful when algorithms, such as neural networks, are sensitive to the magnitude of input features. 
Normalizing ensures that each feature contributes equally to the result, leading to more effective and faster convergence.
```

File: report_thesis/src/sections/background.tex [hunk 5]
```diff
@@ -5,7 +5,18 @@ \subsubsection{Standard Scaler}
...
+\subsubsection{Min-Max Normalization}
+Min-max normalization rescales the range of features to $[0, 1]$ or $[a, b]$. 
+The goal is to normalize the range of the data to a specific scale, typically 0 to 1. 
+Mathematically, min-max normalization is defined as:
+
+$$ 
+	v' = \frac{v - \min(F)}{\max(F) - \min(F)} \times (new max - new min) + new min,
+$$
+
+where $v$ is the original value, $\min(F)$ and $\max(F)$ are the minimum and maximum values of the feature $F$, respectively, and $new min$ and $new max$ define the range of the transformed data.
```
Comment (chhoumann) [inline report_thesis/src/sections/background.tex:17]: also looks bad

> and $new min$ and $new max$ define the range of the transformed data.

File: report_thesis/src/sections/background.tex [hunk 6]
```diff
@@ -5,7 +5,18 @@ \subsubsection{Standard Scaler}
...
+The goal is to normalize the range of the data to a specific scale, typically 0 to 1. 
+Mathematically, min-max normalization is defined as:
+
+$$ 
+	v' = \frac{v - \min(F)}{\max(F) - \min(F)} \times (new max - new min) + new min,
+$$
+
+where $v$ is the original value, $\min(F)$ and $\max(F)$ are the minimum and maximum values of the feature $F$, respectively, and $new min$ and $new max$ define the range of the transformed data. 
+This type of normalization is particularly useful when algorithms, such as neural networks, are sensitive to the magnitude of input features. 
+Normalizing ensures that each feature contributes equally to the result, leading to more effective and faster convergence.
```
Comment (chhoumann) [inline report_thesis/src/sections/background.tex:19]: to which result?

section reads mostly geared towards neural nets / other algs that use gradient descent, no?

File: report_thesis/src/sections/background.tex [hunk 7]
```diff
@@ -5,7 +5,18 @@ \subsubsection{Standard Scaler}
 
 \subsubsection{Max Absolute Scaler}
 
-\subsubsection{MinMax Scaler}
+\subsubsection{Min-Max Normalization}
+Min-max normalization rescales the range of features to $[0, 1]$ or $[a, b]$.
```
Comment (chhoumann) [inline report_thesis/src/sections/background.tex:9]: you use $a$ and $b$ but don't reuse them in the formula below. i'd say that would be better than $new max$ and $new min$.

**Review by chhoumann — CHANGES_REQUESTED** [review]

File: report_thesis/src/sections/background.tex [hunk 8]
```diff
@@ -15,19 +15,16 @@ \subsubsection{Max Absolute Scaler}
...
 $$ 
-	v' = \frac{v - \min(F)}{\max(F) - \min(F)} \times (new max - new min) + new min,
+	v' = \frac{v - \min(F)}{\max(F) - \min(F)} \times (b - a) + a,
 $$
+where $v$ is the original value, $\min(F)$ and $\max(F)$ are the minimum and maximum values of the feature $F$, respectively. 
 
-where $v$ is the original value, $\min(F)$ and $\max(F)$ are the minimum and maximum values of the feature $F$, respectively, and $new min$ and $new max$ define the range of the transformed data. 
-This type of normalization is particularly useful when algorithms, such as neural networks, are sensitive to the magnitude of input features. 
-Normalizing ensures that each feature contributes equally to the result, leading to more effective and faster convergence.
+This type of normalization is beneficial because it ensures that each feature contributes equally to the analysis, regardless of its original scale. This balanced contribution helps improve the performance and convergence speed of many algorithms, making the learning process more effective and efficient.
```
Comment (chhoumann) [inline report_thesis/src/sections/background.tex:27]:
```suggestion
This type of normalization is beneficial because it ensures that each feature contributes equally to the analysis, regardless of its original scale.
```

**Review by chhoumann — APPROVED** [review]
